docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -191,7 +191,120 @@ adding a Channel Binder implementation
 (e.g. `spring-cloud-starter-stream-rabbit` for RabbitMQ or
 `spring-cloud-starter-stream-kafka` for Kafka). This will
 automatically turn your app into a producer of messages with payload
-type `Spans`. 
+type `Spans`.
+
+== Customizations
+
+Thanks to the `SpanInjector` and `SpanExtractor` you can customize the way spans
+are created and propagated.
+
+Essentially with Spring Cloud Span there are two ways span tracing data can be passed to
```

Comment:
There are currently two built-in ways to pass tracing information between processes: